styles/variables-silicon.less

```diff
@@ -8,6 +8,18 @@
 // to this file with the real value for the "silicon" skin.
 //
 
+// Common
+// --------------------------------
+@agate-base-font-family: "MuseoSans", sans-serif;
```

Comment:
Typically what we do is define a CSS font-family in the `styles/internal/fonts.less` file, and associate several fonts to it. Then we can refer to that font family easily in the variables files which allows for better fallback support and maintenance.

I would recommend cloning one of those `font-*` files and updating the names to refer to LGSmartUI local font with the custom name "LG Agate UI" (or something like that), then changing this line to be the following:
```css
@agate-base-font-family: "LG Agate UI", "MuseoSans", sans-serif;
```

By using this approach, if the LGSmartUI font is present and available, it will be used, if not, the MuseoSans font will be used, and if that isn't downloadable for some reason, a generic sans-serif font will be used. SAfe and easy fallbacks.

Additional note, you've referred to the PostScript font name, which is only used by macOS, not by webOS, so you'll need to specify both Postscript name "LGSmartUI" and full name "LG Smart UI", similar to the other fonts defined in Agate.

Additional note 2, you've referred to the family name of the font, which doesn't allow you to be selective about which font-weights map to which CSS font weights. The font has 4 weights, as well as now 2 styles. Some manual set up is required.

Reply:
@stanca-pop-lgp @razvan-jurca-lgp Oh, I missed regarding font yesterday. We will keep using `Museo` not `LG Smart UI` on Agate. GUI guide needs to be updated. 
@Djspaceg Thanks for the heads up :)